LICENSE

```diff
@@ -200,3 +200,30 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 
+----
+
+This product bundles Glyphicon Halflings ( http://glyphicons.com/
+ ) via Bootstrap 3's distributing these under The MIT License.
+
+This product bundles angular-ui / ui-sortable / sortable.js ( https://github.com/angular-ui/ui-sortable/blob/master/src/sortable.js ) under The MIT License.
+
+
+The MIT License (MIT)
```

Comment:
Per the Apache guide, MIT license should not not appended.

> In LICENSE, add a pointer to the dependency's license within the distribution and a short note summarizing its licensing

ref: https://www.apache.org/dev/licensing-howto.html#permissive-deps
ref: https://mail-archives.apache.org/mod_mbox/incubator-general/201203.mbox/%3CCBFE3722-1B5C-48FC-AD82-0931EEC6F143@gbiv.com%3E